events/te_barbarians.txt

```diff
@@ -2429,6 +2431,8 @@ te_barbarians.7 = {
             tag = SEL
             tag = MRY
             tag = BHA
+            tag = PTH #Parthia

```

Comment:
I'd add a scripted trigger to avoid code duplication with te_barbarians.6.

Reply:
Thanks, added!